client/components/boards/boardsList.styl

```diff
@@ -128,3 +125,18 @@ $spaceBetweenTiles = 16px
         font-size: 25px
         color: white
 
+@media screen and (max-width: 800px)
```

Comment:
Maybe we could have a mode with 3 items per row, between the 2 on mobile and 4 on desktop?